ansible/roles/airflow/tasks/af-container.yml

```diff
@@ -0,0 +1,36 @@
+---
+- file: path=/srv/{{ dockname }} state=directory mode=0750 owner={{ login }} group={{ login }}
+- stat: path=/srv/{{ dockname }}/airflow.cfg
+  register: afcfg
+- command: cp -a /srv/etc/af-share-{{ airflow_tag }}/airflow.cfg /srv/{{ dockname }}/airflow.cfg
```

Comment:
```
[copy]
[...]
- remote_src
        If False, it will search for src at originating/master
        machine, if True it will go to the remote/target machine for
        the src. Default is False.
        Currently remote_src does not support recursive copying.
        (Choices: True, False)[Default: no]
[...]
```